atomic_reactor/plugins/pre_fetch_maven_artifacts.py

```diff
@@ -116,34 +116,25 @@ def process_by_url(self, url_requests):
 
     def download_files(self, downloads):
         artifacts_path = os.path.join(self.workdir, self.DOWNLOAD_DIR)
+        koji_config = get_koji(self.workflow)
+        insecure = koji_config.get('insecure_download', False)
 
         self.log.debug('%d files to download', len(downloads))
         session = util.get_retrying_requests_session()
 
         for index, download in enumerate(downloads):
             dest_path = os.path.join(artifacts_path, download.dest)
             dest_dir = dest_path.rsplit('/', 1)[0]
+            dest_filename = dest_path.rsplit('/', 1)[-1]
```

Comment:
we can also just use

os.path.basename
os.path.dirname  

instead those rsplits

Reply:
updated.